### PR TITLE
Allow the receive to log a short message if an event is received.

### PR DESCRIPTION
**Pull request by pilhuhn**
Also emit timing metrics.

Signed-off-by: Heiko W. Rupp <hrupp@redhat.com>

### CODE REVIEW

File: external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java [hunk 1]
```diff
@@ -31,16 +32,23 @@
...
     @Inject
     AlertsService alertsService;
 
+    @Timed(name = "engine-incoming", absolute = true)
     @Incoming("kafka-hosts")
     public CompletionStage<Void> processAsync(JsonObject json) {
         String tenantId = json.getString(TENANT_ID);
         String insightsId = json.getString(INSIGHT_ID_FIELD);
+        if (logEvents) {
+            System.out.printf("Got msg for tenant [%s] and insightsId [%s] \n ",  tenantId , insightsId);
```
Comment (burmanm) [inline external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java]: This should use log.debugf()

File: external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java [hunk 2]
```diff
@@ -31,16 +32,23 @@
     @ConfigProperty(name = "engine.receiver.store-events")
     boolean storeEvents;
 
+    @ConfigProperty(name = "engine.receiver.log-events", defaultValue = "false")
```
Comment (burmanm) [inline external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java]: Lets keep logging properties on logging configuration.

Comment (pilhuhn) [reply external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java]: How do you mean?
The intention here is to switch this on/off from the env in Kube.

File: external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java (hunk 1 above)
Comment (pilhuhn) [reply external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java]: Well. In container world that all goes to stdout, so I am not sure if that is really needed.

Comment (burmanm) [reply external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java]: It is necessary for the behavior of async processing (System.out isn't async).

File: external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java (hunk 2 above)
Comment (burmanm) [reply external/src/main/java/com/redhat/cloud/custompolicies/engine/process/Receiver.java]: You can do that with env parameters using ``quarkus.log.category."categories".level``

**Comment by pilhuhn** [comment]
I am merging this, so that we can use it for debugging purposes. We can still change the way the logging is enabled later (or completely remove that again).

**Pull request merged**